.gitignore

```diff
@@ -12,3 +11,4 @@ docs/
 npm-debug.log
 /.idea/
 .env
+dist/*
```

Comment:
🆗  let's keep this as is and we can change it before the release if we cannot find a way to work this out.